src/frontend/src/btc/services/worker.btc-wallet.services.ts

```diff
@@ -27,16 +37,24 @@ export const initBtcWalletWorker = async ({
 
 	return {
 		start: () => {
+			const isTestnetNetwork = isNetworkIdBTCTestnet(networkId);
+			const isRegtestNetwork = isNetworkIdBTCRegtest(networkId);
+
 			worker.postMessage({
 				msg: 'startBtcWalletTimer',
 				data: {
+					btcAddress: get(
```

Comment:
I would not get the address imperatively, that can lead to some surprises. We should probably stop/start the worker each time the address change and pass it as a variable?

If you want to tackle this in another PR right after this one, we can just add a TODO for now that way we move on? WDYT?